<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>relayPage</class>
 <widget class="QDialog" name="relayPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>465</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_20">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,4">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>显示</string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>18</y>
              <width>451</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>文字：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="textLineEdit"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>178</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>49</y>
              <width>451</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3,1,1,3,4,2">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>颜色：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="textClrBtn">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_14">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>18</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>字体：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="fontBtn">
               <property name="text">
                <string>示例</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>78</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="moreBtn">
               <property name="text">
                <string>更多属性</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>形状</string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>8</x>
              <y>20</y>
              <width>471</width>
              <height>271</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,1,1,2">
             <item>
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>基本</string>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>6</x>
                  <y>50</y>
                  <width>451</width>
                  <height>22</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="2,2,2,1">
                 <item>
                  <widget class="QRadioButton" name="noFillRadio">
                   <property name="text">
                    <string>无填充</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="clrFillRadio">
                   <property name="text">
                    <string>颜色填充</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="picFillRadio">
                   <property name="text">
                    <string>图片填充</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_15">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>108</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>6</x>
                  <y>20</y>
                  <width>451</width>
                  <height>22</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="2,4,1,3">
                 <item>
                  <widget class="QCheckBox" name="seeFrameCheck">
                   <property name="text">
                    <string>边框可见</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_11">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>148</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>边框样式：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="styleComboBox"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_7">
               <property name="title">
                <string>透明度</string>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>7</x>
                  <y>14</y>
                  <width>451</width>
                  <height>22</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <widget class="QSlider" name="transHSlider">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="transSpinBox">
                   <property name="suffix">
                    <string>%</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>填充</string>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>7</x>
                  <y>17</y>
                  <width>441</width>
                  <height>25</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,2,1,2,1">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>填充方式：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="fillStyleCombo"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>颜色：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="fillClrBtn">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_16">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>78</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_10">
               <property name="title">
                <string>图片</string>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>17</y>
                  <width>441</width>
                  <height>25</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="9,1">
                 <item>
                  <widget class="QLineEdit" name="picLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="picBtn">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>49</y>
                  <width>441</width>
                  <height>22</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="2,3,2">
                 <item>
                  <widget class="QCheckBox" name="keepPicCheck">
                   <property name="text">
                    <string>保持图片比例</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_17">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>178</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QComboBox" name="alignPicCombo"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>动作</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>41</y>
               <width>305</width>
               <height>291</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>128</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>操作</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>108</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>操作类型</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QRadioButton" name="openGraphRadio">
                   <property name="text">
                    <string>打开画面</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="picOperRadio">
                   <property name="text">
                    <string>画面操作</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>118</width>
                     <height>17</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>画面选择</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,8,0">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>打开画面：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="openPicCombo"/>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>21</width>
                     <height>17</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>画面操作</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QRadioButton" name="stationConfirmRadio">
                   <property name="text">
                    <string>厂站确认</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="graphComfirmRadio">
                   <property name="text">
                    <string>本图确认</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>118</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="page_2">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>13</x>
               <y>26</y>
               <width>298</width>
               <height>281</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,7,1">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>128</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>遥控</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>128</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>遥控</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,8,1">
                   <property name="spacing">
                    <number>1</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>厂站：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="stationCombo"/>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_8">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>18</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,8,1">
                   <property name="spacing">
                    <number>1</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>遥控点：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="pointCombo"/>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_9">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>18</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,8,1">
                   <property name="spacing">
                    <number>1</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>遥控值：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="valueCombo"/>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_10">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>18</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
